### ISSUE TITLE
Make it possible to change the language of the site without an account

### ISSUE THREAD

**Issue by Bevinsky** [ewhal/nyaa#329]
Currently, the site will pick a language for you based on what your browser specifies. It's possible to change it if you have an account and are logged in, but it's still a bit annoying that you can't change it without one.

My suggestion is a language dropdown in the footer or something that adds a cookie with a language. Priority of language selection goes account -> cookie -> browser header.

**Comment by nixxquality** [comment]
It already supports this, but there is no interface for it:
```Go
	cookie, err := r.Cookie("lang")
	cookieLanguage := ""
	if err == nil {
		cookieLanguage = cookie.Value
	}
```
https://github.com/ewhal/nyaa/blob/86dcac2/util/languages/translation.go#L70-L74

For now, you can open your browser's Javascript debug console (F12) and enter the following command: `document.cookie = "lang=ja-jp"` where ja-jp is the [identifier for the language you want](https://github.com/ewhal/nyaa/tree/master/translations).

**Comment by cosmo321** [comment]
It should be an opt-in feature, not an opt-out feature. The automatic translation to my native language is embarrassingly bad and very awkward to read. For example is "filter remakes" translated to a phrase more similar to something in the vain of "filter retarded torrents". It's awful. 

I shouldn't have to register to have the site presented in a proper way.

**Comment by toynbeeidea** [comment]
>implying remakes aren't retarded

**Comment by akuma06** [comment]
Should be fixed by #407
